Apps/CocoaPods-FCM/src/View/Widget/EnvironmentText.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import SwiftUI
+
+struct EnvironmentText: View {
+    var body: some View {
+        Text("SDK: \(EnvironmentUtil.cioSdkVersion) \n app: \(EnvironmentUtil.appBuildVersion) (\(EnvironmentUtil.appBuildNumber))")
+            .multilineTextAlignment(.center)
+            .foregroundColor(.gray)
+    }
```

Comment:
```suggestion
var body: some View {
        VStack(spacing: 10) {
            Text("App Name: \(EnvironmentUtil.appName)")
            Text("SDK Version: \(EnvironmentUtil.cioSdkVersion)")
            Text("Build Version: \(EnvironmentUtil.appBuildVersion) (\(EnvironmentUtil.appBuildNumber))")
        }
        .multilineTextAlignment(.center)
        .foregroundColor(.gray)
        .padding(10)
    }
```

From my experience working with declarative UI, it's better practice to stack the views rather than use `\n` because the latter approach ends up spacing issues and on different screen sizes it doesn't always stretch/expend as expected.